.github/workflows/CI.yml

```diff
@@ -156,6 +156,25 @@ jobs:
         working-directory: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test
         run: |
           docker exec $(docker ps --filter "name=migration-console" -q) pipenv run pytest /root/lib/integ_test/integ_test/replayer_tests.py --unique_id="testindex" -s
+      - name: Collect Docker, OpenSearch Benchmark, and Shared Logs
+        if: always()
+        run: |
+          mkdir -p logs/docker logs/opensearch_benchmark_logs logs/shared_logs_output
+          for container in $(docker ps -aq); do
+            container_name=$(docker inspect --format '{{.Name}}' $container | sed 's/\///')
+            docker logs $container > logs/docker/${container_name}_logs.txt 2>&1
+          done
+          docker cp $(docker ps --filter "name=migration-console" -q):/root/.benchmark/logs logs/opensearch_benchmark_logs
+          docker cp $(docker ps --filter "name=migration-console" -q):/shared-logs-output logs/shared_logs_output
```

Comment:
This is nice.  Can we create a JIRA so that the migration console has a way to do the log collection for troubleshooting prior to a stack teardown / docker teardown in place of this GHA?

Reply:
https://opensearch.atlassian.net/browse/MIGRATIONS-1990